### PR TITLE
Minor additions to the documentation

### LINKED ISSUE #973: [Bug] Prerequisite `readelf` and `file` should be listed in document explicitly for mac users
### GEF+GDB version

```shell
gef➤  version
GEF: (Standalone)
Blob Hash(/Users/***/.gdbinit-gef.py): 1a409c397646df9c0b8549d576cd7c7f8ec1ef47
SHA256(/Users/***/.gdbinit-gef.py): f096ff3bf50df08dbbc698a919360014f86f1eea315011cdabebd72a30a5bce8
GDB: 13.2
GDB-Python: 3.11
```

### Operating System

MacOS (Intel)

### Describe the issue you encountered

By following the installation manual, mac users may encounter `missing file` errors:

```python
...
 File "/Users/***/.gdbinit-gef.py", line 1843, in which
    raise FileNotFoundError(f"Missing file `{program}`")
FileNotFoundError: Missing file `readelf` 
```

Also there is another report  #911.

### Do you read the docs and look at previously closed issues/PRs for similar cases?

Yes

### Architecture impacted

- [ ] X86
- [X] X64
- [ ] ARM
- [ ] ARM64
- [ ] MIPS
- [ ] MIPS64
- [ ] PPC
- [ ] PPC64
- [ ] RISCV

### Describe your issue. Without a proper reproduction step-by-step, your issue will be ignored.

I checked the source and found a special handling adapting to mac:
https://github.com/hugsy/gef/blob/0461d6fe6e3592f333e11ab39947905171ba9011/gef.py#L3315-L3317

So I suppose this author had made tests on mac and proven its workability.

----

`file` and `readelf` may be installed through `homebrew`: 
```bash
$ brew install file binutils
```

And since `binutils` is "keg-only", one has to add its bin path into `$PATH` before starting `gdb/gef`:

```bash
$ export PATH="/usr/local/opt/binutils/bin:$PATH"
$ gdb
```

After that `gef` looks working good:
```
(remote) gef➤  version
GEF: (Standalone)
Blob Hash(/Users/***/.gdbinit-gef.py): 1a409c397646df9c0b8549d576cd7c7f8ec1ef47
SHA256(/Users/***/.gdbinit-gef.py): f096ff3bf50df08dbbc698a919360014f86f1eea315011cdabebd72a30a5bce8
GDB: 13.2
GDB-Python: 3.11
(remote) gef➤  gef-remote
[!] You already are in remote session. Close it first before opening a new one...
```

The document should add some content about setting up `gef` on macos.

### Minimalist test case

_No response_

### Additional context?

_No response_

### PR DESCRIPTION
**Pull request by hugsy**
## Description/Motivation/Screenshots

 - Added more explicit references to gef-extras
 - Added a note about required binaries (fixes #973)

## Against which architecture was this tested ?

Not applicable

## Checklist

- [x] My PR was done against the `dev` branch, not `main`.
- [x] My code follows the code style of this project.
- [x] My change includes a change to the documentation, if required.
- [x] If my change adds new code, [adequate tests](docs/testing.md) have been added.
- [x] I have read and agree to the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by therealdreg** [comment]
For me its ok! :-D

**Review by therealdreg — APPROVED** [review]

File: docs/install.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 ## Prerequisites
 
+Specific GEF commands rely on commonly used Unix commands to extract additional information. Therefore it requires the following binaries to be present:
+  * `file`
+  * `readelf`
+  * `ps`
+  * `python3`
```
Comment (Grazfather) [inline docs/install.md:9]:
```suggestion
Specific GEF commands rely on commonly used Unix commands to extract additional information.
Therefore it requires the following binaries to be present:
- `file`
- `readelf`
- `ps`
- `python3`
```

File: docs/install.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+
+Those tools are included by defautl in many modern distributions. If they're missing, you can use your OS package manager to install them.
```
Comment (Grazfather) [inline docs/install.md:11]:
```suggestion
Those tools are included by default in many modern distributions. If they're missing, you can use your OS package manager to install them.
```

(typo + wrap)

File: docs/install.md [hunk 3]
```diff
@@ -2,6 +2,14 @@
 
 ## Prerequisites
 
+Specific GEF commands rely on commonly used Unix commands to extract additional information. Therefore it requires the following binaries to be present:
+  * `file`
+  * `readelf`
+  * `ps`
+  * `python3`
+
+Those tools are included by default in many modern distributions. If they're missing, you can use your OS package manager to install them.
```
Comment (Grazfather) [inline docs/install.md:11]:
```suggestion
Those tools are included by default in many modern distributions. If they're missing, you can use
your OS package manager to install them.
```

File: docs/install.md [hunk 4]
```diff
@@ -2,6 +2,15 @@
 
 ## Prerequisites
 
+Specific GEF commands rely on commonly used Unix commands to extract additional information. Therefore it requires the following binaries to be present:
```
Comment (Grazfather) [inline docs/install.md:5]: Seems the wrap isn't applied

**Pull request merged**